org.mixedrealitytoolkit.uxcore/Button/PressableButton.cs

```diff
@@ -640,38 +640,36 @@ private float IntersectQuad(Vector3 start, Vector3 end, float depth)
         }
 
         /// <summary>
-        /// Registers the duple Collider + XRBaseInteractor as triggering proximity.
+        /// Registers the interactable triggering proximity.
```

Comment:
"NearInteractionModeDetector" instead of "Interactable"?

Reply:
Went with 'Detector' in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/659/commits/655e552cbcb02e2bdf86e6af4a0a52bebf58f033, resolving.